doc/usersguide/codes/openmc.rst

```diff
@@ -0,0 +1,134 @@
+a specific naming convention is required for OpenMC. To define materials,
+the material ID must be provided in the group name. The format for the group
+name is as follows: ``mat:[matid]``.
+
+``[matid]`` should be replaced by the material ID that will be specified in the
+``materials.xml`` file.
+
+For example, consider a problem where material 7 should be assigned to volumes 4
+through 18. The following command should be used to specify that information:
+::
+
+    CUBIT> group "mat:7" add vol 4 to 18
+
+All volumes must belong to a group, if you want volumes to be filled with vacuum
+then add them to a group called, "mat:Vacuum":
+::
+
+    CUBIT> group "mat:Vacuum" add vol 4 to 18
+
+Surface boundary conditions
+----------------------------
+
+The following boundary conditions available in OpenMC are supported in
+DAGMC. These are:
+
+- vacuum
+- reflective
+- transmission
+
+For example, suppose you want to specify that surfaces 10 and 11 should be
+reflecting surfaces. This command would achieve that:
+::
+
+    CUBIT> group "boundary:Reflecting" add surf 10 11
+
+**Note: surfaces without a specified boundary condition will be set to ``transmission``.**
+
+Problem boundary
+----------------
+
+The DAGMC model should have a "containing volume" which bounds the volumes of
+interest. This volume represents a particle "graveyard" or a region where
+particles are removed from the simulation upon entry. This volume represents the
+boundary between the problem and the outside world.  This volume should surround
+the entire geometry with a shell of finite thickness. Any geometric shape can be
+used, but a cubic shell is recommended to maximize performance.
+
+To create a containing volume, make two volumes in Cubit/Trelis with the same
+shape and same center with one slightly larger than the other, making sure that
+both bound the entire problem geometry. Then, subtract the smaller one from the
+larger one. The result is a containing volume for the problem.
+
+For example, consider a geometry containing 99 volumes, all of which fit inside
+a cube of side length 99 cm centered at the origin. The following commands would
+create a valid graveyard for this problem:
+::
+
+    CUBIT> create brick x 100             # This will be volume 100
+    CUBIT> create brick x 105             # This will be volume 101
+    CUBIT> subtract vol 100 from vol 101  # This will produce volume 102
+    CUBIT> group "mat:Graveyard" add volume 102
+
+A volume in the ``mat:Graveyard`` group will be assigned a void material and its
+surfaces will be given vacuum boundary conditions when the OpenMC simulation is
+initialized.
+
+Temperatures
+------------
+
+Volume temperatures can be defined in OpenMC using a similar syntax to materials or boundary
+conditions but with "temp" as the keyword.
+
+To assign a temperature of 900K to a volume one can use the following command.
+::
+    CUBIT> group "temp:900" add vol x
```

Comment:
There needs to be a blank line between the `::` line and the `CUBIT>` line

Reply:
This was not resolved

Reply:
I guess now that I look closer, this only results in a warning, not an error. But I feel it's still good to not introduce warning messages if we don't need to.

Reply:
Whoops, my bad. Updating..

Reply:
Done.